resources/lang/en/general.php

```diff
@@ -88,6 +88,11 @@
     'firstname_lastname_underscore_format' => 'First Name Last Name (jane_smith@example.com)',
     'lastnamefirstinitial_format' =>  'Last Name First Initial (smithj@example.com)',
     'first'					=> 'First',
+    'firstnamelastname'     => 'First Name Last Name (janesmith@example.com)',
+    'lastname_firstinitial' => 'Last Name First Initial (smith_f@example.com)',
```

Comment:
Same here, since we're using Jane Smith as the dummy name. The `f` might be confusing for folks

Reply:
typo corrected